### PR TITLE
Poc - Person presence indicator

### PR DESCRIPTION
**Pull request by eskil0312**

### CODE REVIEW

File: src/components/org/OrgChart/OrgChart.stories.tsx [hunk 1]
```diff
@@ -154,7 +154,42 @@ const position: Position = {
                     name: 'Equinor',
                 },
                 contracts: [],
-                positions: [],
+                positions: [
```
Comment (eskil0312) [inline src/components/org/OrgChart/OrgChart.stories.tsx:157]: Positions property was added for testing right? If that's the case, we don't need it

Comment (ingridklepsvik) [reply src/components/org/OrgChart/OrgChart.stories.tsx]: Removed it

File: src/components/people/PersonDetail/index.tsx [hunk 2]
```diff
@@ -1,54 +1,132 @@
...
+
+        try {
+            const response = await apiClients.people.getPresenceAsync(mail, '1.0-preview');
+            setPresence(response.data);
+        } catch (e) {
+            setPresenceError(e);
+        } finally {
+            setIsFetchingPresence(false);
+        }
+    }, []);
```
Comment (eskil0312) [inline src/components/people/PersonDetail/index.tsx:46]: Maybe add apiClients to dependency array?

File: src/components/people/PersonPhoto/__stories__/PersonPhoto.stories.tsx [hunk 3]
```diff
@@ -73,6 +73,14 @@ const sizes = {
     small: 'small',
 };
 
+/* const presenceStatus = {
+    available: 'available',
+    away: 'away',
+    doNotDisturb: 'doNotDisturb',
```
Comment (eskil0312) [inline src/components/people/PersonPhoto/__stories__/PersonPhoto.stories.tsx:79]: Remove comment?

File: src/components/people/PersonDetail/index.tsx (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/people/PersonDetail/index.tsx]: Added apiClients to dependency array

File: src/components/people/PersonPhoto/__stories__/PersonPhoto.stories.tsx (hunk 3 above)
Comment (ingridklepsvik) [reply src/components/people/PersonPhoto/__stories__/PersonPhoto.stories.tsx]: Removed comment

**Pull request merged**